### DISCUSSION TITLE
how to download to sdcard instaed of internal storage?

### DISCUSSION THREAD

**Discussion by KraXen72** [z-huang/music#1089] [Q&A]
how can i download to sd card instead of internal storage? i made 2 issues about this, but they both got deleted. why is that?

**Comment by z-huang** [comment]
> You should ensure the completion of the task before proceeding to check it off the checklist. Neglecting to do so may impede the efficiency of the issue resolution process. The developer has the right to delete the issue directly if you check the list blindly.
> - [ ] I've checked that there is no other issue about this feature request.

I highly recommend that you meticulously adhere to the checklist, particularly the step that requires verifying the absence of any duplicate issues pertaining to this feature request. It has come to my attention that there are numerous duplicate issues already in existence, namely #1049, #987, #1040, and #1046. It is crucial not to overlook this checklist item, as doing so would not only be a disservice to our time but also to the overall issue management process.

**Reply by KraXen72** [reply]
sorry, i searched 'sd card' in issues, but i forgot to check closed issues. all those issues only say duplicate of #987, but #987 provides no reasoning why you decided not to implement this. i don't care about opening the chunks, i just want to only have them on sd card, so i can download over 10gb of songs without worry. 
What is the reason chunks cannot be stored & read from sd card? 
Please re-consider implementing this, as even you showed me several people requested this and it would be really appreciated by many users. Thanks!

**Reply by KraXen72** [reply]
bump
